Src/Gtk2/DasherAppSettings.cpp

```diff
@@ -153,3 +153,11 @@ void option_help()
     g_print("%-30s %-12s  %s\n", stringparamtable[i].regName, stringparamtable[i].defaultValue, stringparamtable[i].humanReadable);
   }
 }
+
+void option_version()
+{
+  g_print("\n");
+  // Note to translators: This is the version keyword showing for the command line option "--version"
+  g_print("%-30s %-30s %-30s\n", _("Version"), PACKAGE_VERSION, _("License GPL 2+"));
```

Comment:
You should leave "GPL 2+" out of the translation.

Also, you should add a note to the translators for 'License".

So I think it makes sense to put Version and License in two consecutive g_prints instead of a single one. I think It would also visually look better.